src/Wallabag/CoreBundle/Tests/Controller/EntryControllerTest.php

```diff
@@ -19,6 +19,36 @@ public function testLogin()
         $this->assertContains('login', $client->getResponse()->headers->get('location'));
     }
 
+    public function testQuickstart()
+    {
+        $this->logInAs('empty');
+        $client = $this->getClient();
+
+        $client->request('GET', '/unread/list');
+        $client->followRedirect();
+
+        $this->assertEquals(200, $client->getResponse()->getStatusCode());
+        $this->assertContains('We\'ll accompany you to visit wallabag', $client->getResponse()->getContent());
+
+        // Test if quickstart is disabled when user has 1 entry
+        $crawler = $client->request('GET', '/new');
+
+        $this->assertEquals(200, $client->getResponse()->getStatusCode());
+
+        $form = $crawler->filter('button[type=submit]')->form();
+
+        $data = array(
+            'entry[url]' => 'https://www.wallabag.org/blog/2016/01/08/wallabag-alpha1-v2',
```

Comment:
If I use `$this->url` here, I've got this error further, and I don't know why (`arpenter` is in article content):

```
1) Wallabag\CoreBundle\Tests\Controller\EntryControllerTest::testPostNewOk
Failed asserting that 'arpenter' contains "Google".
/home/nicosomb/dev/wallabag/v2/src/Wallabag/CoreBundle/Tests/Controller/EntryControllerTest.php:134
```
Related to #1566?